nn_sdwan/README.md

```diff
@@ -0,0 +1,61 @@
+# Agent Check: nn-sdwan
+
+## Overview
+
+This check monitors SDWAN controllers using a SDWAN platorm provided by [Netnology][1] through the Datadog Agent. Currently only Cisco vManage devices are supported.
```

Comment:
```suggestion
This check monitors SDWAN controllers through the Datadog Agent using a SDWAN platorm provided by [Netnology][1]. Currently only Cisco vManage devices are supported.
```

This might be a bit clearer?